index.html

```diff
@@ -248,6 +250,14 @@ <h2>Glossary</h2>
 
 <div class="letter_anchor" id="i">I</div>
 
+<p><dfn id="def_ijam" class="lint-ignore export">Ijam</dfn>. An ijam is an Arabic script diacritical mark considered to be part of a basic letter form. &#x681; [<span class="uname" translate="no">U+0681 ARABIC LETTER HAH WITH HAMZA ABOVE</span>] is an example of a letter with an ijam. Unicode encodes letter+ijam combinations as separate, atomic characters, which are never given decompositions in the standard. Ijam generally take the form of one-, two-, three- or four-dot markings above or below the basic letter skeleton, although other diacritic forms occur in extensions of the Arabic script in Central and South Asia and in Africa.</p>
+
+<aside class="note">
```

Comment:
The glossary keeps definitions short on purpose, and limits the descriptions to one paragraph.  For this kind of additional information (some of which actually begins to wander off track) we should point out to somewhere else (probably your original source for this text, which is https://r12a.github.io/scripts/arabic/arb.html#ijam_tashkil).

Reply:
The definition really must have a link to the tashkil entry, and vice versa.

Reply:
The example of hah+hamza needs clarification to indicate that it represents a consonant in Pashto.  I will add that to the orthography description too.